DatadogRUM/Sources/RUMMonitorProtocol.swift

```diff
@@ -113,8 +113,9 @@ public protocol RUMMonitorProtocol: AnyObject {
     /// This method should be called only once per view.
     /// If the view is not started, this method does nothing.
     /// If the view is not active, this method does nothing.
+    /// - Parameter overwrite: if true, overwrites the previosuly calculated view loading time.
     @_spi(Experimental)
-    func addViewLoadingTime()
+    func addViewLoadingTime(overwrite: Bool)
```

Comment:
question/ Is it conscious decision to make this param required? (no strong opinion on this, just checking)

Reply:
Yes, that's intentional. We don't want customers to explicitly opt-in what behavior they want.